### ISSUE TITLE
Better alt text for images

### ISSUE THREAD

**Issue by tbooth** [carpentries-incubator/snakemake-novice-bioinformatics#41]
From @jdblischak :

I confirmed that the main figures had alt text. They could probably use more though if the goal is to convey the same information to someone using a screen reader. For example, the information contained in the image at the beginning of [Episode 7](https://carpentries-incubator.github.io/snakemake-novice-bioinformatics/07-finished-workflow/index.html) summarizes the input files (eg 18 in total due to paired reads, 3 conditions, and 3 reps) and the various steps. The steps are well described in the text above, but not the input files.

**Comment by tbooth** [comment]
Here are the alt texts in the episodes, as per a quick grep:

```
$ grep '\!\[' _episodes/*.md
_episodes/03-chaining_rules.md:![visual representation of the above process with arrows showing the order wildcards and
_episodes/04-the_dag.md:![diagram showing jobs as coloured boxes joined by arrows representing data flow][fig-dag]
_episodes/04-the_dag.md:![DAG for partial workflow with four boxes. The boxes for kallisto_index and trimreads are dotted,
_episodes/06-awkward_programs.md:![Screenshot of a typical FastQC report specifically showing the per-base quality box-and-whisker
_episodes/07-finished-workflow.md:![Summary of our full QC workflow with icons representing the steps listed above][fig-workflow]
_episodes/09-performance.md:![Representation of a computer with 4 cores and 5 jobs wanting 1, 1, 1, 2 and
_episodes/09-performance.md:![A photo of some high performance compute arranged in five racks in a server room][fig-cluster]
_episodes/11-assembly_challenge.md:![box-and-arrow representation of the cutadapt, concatenation, assembly and summary steps to
```

8 figures in 6 episodes.

**Comment by tbooth** [comment]
I've added a lot more text to describe the images. Bearing in mind that this should be suitable for those using a screen reader, I've tried to describe what the image looks like - eg. "there is a box labelled cutadapt with an arrow  pointing down to another box" -rather than what the image is trying to convey  - eg. "there is a cutadapt job with output going to the next job".

There is probably room for further improvement, but I think this issue is now addressed.
